client/layout/guided-tours/tours/editor-insert-menu-tour.js

```diff
@@ -36,8 +38,19 @@ class RepositioningStep extends Step {
 		clearInterval( this.interval );
 	}
 
+	render() {
+		if ( this.props.isEditorModeVisual ) {
+			return super.render();
+		}
+		return null;
+	}
+
 }
 
+const ConnectedStep = connect( state => ( {
+	isEditorModeVisual: 'html' !== getPreference( state, 'editor-mode' ),
```

Comment:
The idea was that we could one day change the visual editor mode from `tinymce` to something else, while likely the HTML mode will stay `html`.
Since this GT has been currently pulled for buggy behaviour (https://github.com/Automattic/wp-calypso/pull/10706), I'm not sure my long-term reasoning makes much sense.